migrations/20230313212225_metadata.sql

```diff
@@ -0,0 +1,6 @@
+create table if not exists metadata (
```

Comment:
Let's have instead a table `package_indexes` with the following:

* package_index uuid primary key;
* repository text not null;
* timestamp timestamptz not null;

And an index on the repository field.

:-)

Reply:
Does having a repository make sense quite yet? At some point we'll want to point to a repo and then will make sense, but right now we're importing from a directory or an index file, which doesn't have a repository attached to it. Maybe it makes the most sense to default to `hackage.haskell.org` and perhaps have an argument to the mirror-cli for specifying the repository?

Reply:
Good question, at the moment the only repo is Hackage so you can default to it (in the Haskell code, not the SQL). 
A later workflow for CHaP will have to make the distinction between the two, but we're in the clear for now :)